shared/libraries/config_protocol/include/config_protocol/config_client_device_impl.h

```diff
@@ -218,9 +218,13 @@ void GenericConfigClientDeviceImpl<TDeviceBase>::onRemoteUpdate(const Serialized
         if (!serialized.hasKey(id))
         {
             if (this->defaultComponents.count(id))
-                throw InvalidOperationException{"Serialized remote object does not contain default device component: " + id};
-            
-            toRemove.push_back(id);
+            {
+                DAQLOGF_W(this->loggerComponent, "The server does not provide default device component: {}", id);
```

Comment:
I don't think we should be outputting a warning for every device that was created with an older version of openDAQ. This is likely a debug message.